cmd/zc_enumerator.go

```diff
@@ -587,14 +587,14 @@ type preFilterProvider interface {
 type syncEnumerator struct {
```

Comment:
I think these enumerators should be useful too.

Reply:
Sync Enumerator may be not , we are trying to scan source only on the basis of Last modified time. As per initial thought we don't want to scan destination, try to keep some meta-data on premise. Which help in finding delta.